go.sum

```diff
@@ -605,8 +603,8 @@ github.com/onsi/ginkgo v1.11.0 h1:JAKSXpt1YjtLA7YpPiqO9ss6sNXEsPfSGdwN0UHqzrw=
 github.com/onsi/ginkgo v1.11.0/go.mod h1:lLunBs/Ym6LB5Z9jYTR76FiuTmxDTDusOGeTQH+WWjE=
```

Comment:
We can remove this line and other unwanted versions of packages which this PR IS upgrading

Reply:
I have executed `go mod tidy`, those entries are automatically getting added back to go.sum file.